Resources/public/script/httplug.js

```diff
@@ -0,0 +1,132 @@
+
+
+/**
+ * Toggle hide/show on the message body
+ */
+function httplug_toggleBody(el) {
+  var bodies = document.querySelectorAll(".httplug-http-body");
```

Comment:
Im not sure what you mean. I indent all my JS with 2 spaces. Isn't that consistent? PHP is a completely different language...